### DISCUSSION TITLE
Alternate Success Markup from a Template or Existing Div after AJAX Submit

### DISCUSSION THREAD

**Discussion by TUSAB-web** [solspace/craft-freeform#80] [Questions & Answers] [answered]
I am looking at [this page in the docs](https://docs.solspace.com/craft/freeform/v3/guides/ajax-return-loads-success.html#alternate-success-markup-from-a-template-or-existing-div-after-ajax-submit): 

If I have 2 forms on one page, would I have the script on the bottom repeated with the different div IDs, or just add another line of the div IDs into the same script... referring to this line specifically:
`$('#form-wrapper').html($('#custom-content').html());`

**Comment by gustavs-gutmanis** [comment, marked as answer]
If both your forms need to show the same div's contents, can get by with a single script for both.

First off - change the wrapper to have no ID specified at all

```twig
<div>
    {{ craft.freeform.form('myContactForm').render }}
</div>
```

Then modify the script to have this code:
_Notice that I changed the `document.on` to `document.addEventListener`. The docs will be updated with this change very soon_ 

```html
<script type="text/javascript">
    document.addEventListener('freeform-render-success', function (event) {
        event.preventDefault();

        var form = event.form;

        $(form).parent().html($('#custom-content').html());
    });
</script>
```
